{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/public/views.py

```diff
@@ -75,3 +83,97 @@ def about():
     """About page."""
     form = LoginForm(request.form)
     return render_template("public/about.html", form=form)
+{% if cookiecutter.use_google_logging == "yes" %}
+
+def get_google_provider_cfg():
+    """Returns Google Provide configuration."""
+    return requests.get(current_app.config["GOOGLE_DISCOVERY_URL"]).json()
+
+
+@blueprint.route("/login")
+def login():
+    """Google Login view."""
+    client = WebApplicationClient(current_app.config["GOOGLE_CLIENT_ID"])
+    # Find out what URL to hit for Google login
+    google_provider_cfg = get_google_provider_cfg()
+    authorization_endpoint = google_provider_cfg["authorization_endpoint"]
+
+    # Use library to construct the request for Google login and provide
+    # scopes that let you retrieve user's profile from Google
+    request_uri = client.prepare_request_uri(
```

Comment:
For purposes of security, we should be passing a `state` value here: [docs](https://oauthlib.readthedocs.io/en/latest/oauth2/clients/webapplicationclient.html#oauthlib.oauth2.WebApplicationClient.prepare_request_uri)